src/_gettsim/transfers/kindergeld.py

```diff
@@ -174,3 +176,39 @@ def kind_bis_10_mit_kindergeld(
     """
     out = kindergeld_anspruch and (alter <= 10)
     return out
+
+
+@policy_info(skip_vectorization=True)
+def gleiche_fg_kindergeldempfänger_kind(
+    p_id: numpy.ndarray[int],
+    p_id_kindergeld_empf: numpy.ndarray[int],
+    fg_id: numpy.ndarray[int],
+) -> numpy.ndarray[bool]:
+    """The child's Kindergeldempfänger is in the same Familiengemeinschaft.
+
+    Parameters
+    ----------
+    p_id
+        See basic input variable :ref:`p_id <p_id>`.
+    p_id_kindergeld_empf
+        See basic input variable :ref:`p_id_kindergeld_empf <p_id_kindergeld_empf>`.
+    fg_id
+        See basic input variable :ref:`fg_id <fg_id>`.
+
+    Returns
+    -------
+
+    """
+    fg_id_kindergeldempfänger = join_numpy(
+        p_id_kindergeld_empf,
+        p_id,
+        fg_id,
+        value_if_foreign_key_is_missing=-1,
+    )
+
+    out = [
+        fg_id_empf == fg_id_child if fg_id_empf >= 0 else False
```

Comment:
`if fg_id_empf >= 0` this part is important because there are individual for whom `p_id_kindergeld_empf` is `-1`. Then, `fg_id_kindergeldempfänger` is `-1`.

We could use `np.where` if you like that more? But I figured it would be less readable.

Reply:
I don't get it. `fg_id_empf` == `fg_id_kindergeldempfänger` in that snippet? How can it be -1 and >= 0 at the same time?

Reply:
Ah true, you were already one step ahead 😄 Then `out = fg_id_kindergeldempfänger == fg_id` is sufficient.

I think `fg_id_numpy` can never return `-1`. But we may want to check the input data. We can do this together with #767 and the other issues regarding consistency checks.